### DISCUSSION TITLE
Crashing of Valheim.

### DISCUSSION THREAD

**Discussion by Kha-kis** [mbround18/valheim-docker#142] [Help!] [answered]
I am experiencing crashing of the server within a minute of logging in.

https://pastebin.com/u8etQDBB

This just started today after performing a pull of the image to the update that came out today. However I did not see any major change.

The container has been running for a week no issues.

Not sure what else to look at here.

**Comment by mbround18** [comment, marked as answer]
This looks like you have a corrupted world, try to reach out in the Valheim discord under server help
https://discord.com/invite/44qXMJH

**Reply by Kha-kis** [reply]
Restoring the world file fixed the issue.

Thanks!
